### PR TITLE
Merge release/0.4.0 changes to develop

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: docs/sphinx/raja_license.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@
 RAJA License
 ===================================
 
-RAJA version 0.3.1
+RAJA version 0.4.0
```
Comment (rhornung67) [inline docs/sphinx/raja_license.rst]: We should update the copyright date here to match the stuff in other files; i.e., 2016-2017.  Having a sed script to easily propagate these change throughout all files would be good to add.

Comment (davidbeckingsale) [reply docs/sphinx/raja_license.rst]: Good idea!

**Comment by davidbeckingsale** [comment]
@rhornung67 can I get an approval for this?

**Comment by davidbeckingsale** [comment]
Superseded by #348

**Comment by rhornung67** [comment]
@davidbeckingsale this branch has unmerged commits. Do we need these or can the branch be deleted?

**Comment by davidbeckingsale** [comment]
Deleted.

**Pull request closed without merging**